src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -148,10 +150,58 @@ def init_db():
     """
     # SyncD database connector. THIS MUST BE INITIALIZED ON A PER-THREAD BASIS.
     # Redis PubSub objects (such as those within swsssdk) are NOT thread-safe.
-    db_conn = SonicV2Connector(**redis_kwargs)
+    db_conn = SonicV2Connector(**redis_kwargs) 
 
     return db_conn
 
+def init_multi_db():
+    db_conn= []
+    if (os.path.isfile('/usr/share/sonic/platform/asic.conf')):
+        fp = open('/usr/share/sonic/platform/asic.conf', 'r')
```

Comment:
>/usr/share/sonic/platform/asic.conf [](start = 19, length = 35)

Consider if other component also want to check num_asic, embed this magic string in code is difficult to maintain. How about create a command line argument to feed this parameter, even feed a global database config or a local namespace config.

Reply:
Changed this.